README.md

```diff
@@ -666,6 +666,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Calista Ifenkwe](https://github.com/TechyStarr)
   - [Carlie Hope](https://github.com/carliemaria)
   - [Carmen Chow](https://github.com/cchow33)
+  - [Carlos Herrera](https://github.com/cahema)
```

Comment:
I believe your name must be one line above (`L` comes first and later on `M` does).
Let us know if you need any help updating that. 🙂